tool/plugin/test/plugin_test.dart

```diff
@@ -120,6 +120,16 @@ void main() {
       var loc = content.indexOf('@');
       expect(loc, -1);
     });
+    test('provision', () async {
+      var runner = makeTestRunner();
+      TestBuildCommand cmd;
+      await runner.run(["-d../..", "build"]).whenComplete(() {
```

Comment:
you intend `-d../..` and not something like `-d=../..`?

Reply:
Nevermind, looks like `-dfoo` is correct, and `--cwd=foo`.

Reply:
I added a check that makes the '=' allowable but the args parser expects none. If you use the '--' form then it wants the '=' too. I think I use both forms in test code, intentionally. See the tests named 'test' and 'deploy'.